manifest.webapp

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Files",
+  "slug": "cozy-files-v3",
+  "icon": "app-icon.svg",
+  "description": "File manager for Cozy v3",
+  "source": "https://github.com/cozy/cozy-files-v3.git@build",
+  "developer": {
+    "name": "Cozy",
+    "url": "https://cozy.io"
+  },
+  "default_locale": "en",
+  "locales": {
+    "fr": {
+      "description": "Gestionnaire de fichiers pour Cozy v3"
+    }
```

Comment:
@jsilvestre there is already an english version. It's the default version, line 5